README.md

```diff
@@ -66,10 +66,17 @@ SSH_KEY - Override the default public key built into the operator for public key
 
 #### Initial Setup
 ```bash
-kind create cluster
+kind create cluster --config deploy/cert.yaml
```

Comment:
deploy/cert.yaml is a kind config for creating a cluster with webhooks enabled. For clarity it would be helpful for the file to have a more appropriate name.